### ISSUE TITLE
No execution of commands whilst resolver is active?

### ISSUE THREAD

**Issue by eboracus** [atom/keybinding-resolver#26]
As it stands it seems that pressing a key-binding whilst using the resolver still executes the associated command for that binding. Wouldn't it be better if the actual execution of a given command was blocked whilst the keybinding resolver is active? So just the information for a particular key combination is given, but no actions are taken at that stage.

I've had some pretty inconvenient commands be activated whilst just quickly needing to look at what command (if any) is on a particular key combination. Would it not make sense to have the opportunity to decide whether or not to actually use that command only _after_ getting to see what it is?

**Comment by 50Wliu** [comment]
This sounds like a nice config option.

**Comment by eboracus** [comment]
Yes just came back to edit, to say that maybe an optimal solution would actually be to have a toggle button on the keybinding resolver UI (I guess similar to the find/replace UI toggles) to switch between active/non-active mode. Giving greater flexibility than say an option in the package settings?

**Comment by j-walker23** [comment]
I want this so so so so bad. I have spent hours upon hours on just god damn keybindings. Embarrassing.

If something had your suggestion, along with showing a previous one as well. I could actually compare the two i am trying to resolve.
